src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Color color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Move Random and ColorSupplier to the class field
And move 100 to the class constant
An example can be seen in the checklist for the task.

Reply:
Thank you for your feedback.
Committed.
Please review. 
Thanks!